renderdoc/driver/d3d12/d3d12_device_wrap.cpp

```diff
@@ -2132,10 +2132,183 @@ void WrappedID3D12Device::CopyDescriptorsSimple(UINT NumDescriptors,
   }
 }
 
+template <typename SerialiserType>
+bool WrappedID3D12Device::Serialise_OpenSharedHandle(SerialiserType &ser, HANDLE NTHandle, REFIID riid, void **ppvObj)
+{
+  ID3D12DeviceChild *devChild = ser.IsWriting() ? (ID3D12DeviceChild *)*ppvObj : NULL;
+
+  SERIALISE_ELEMENT_LOCAL(ResourceRIID, riid);
+  SERIALISE_ELEMENT_LOCAL(ResourceId, GetResID(devChild));
```

Comment:
`ResourceId` is the name of the type so using it as a variable name is rather strange. The convention for serialised arguments is to use something similar to the argument name if the argument isn't serialised in-place, even if the type doesn't match up to what's expected, so for this maybe `pObj` would be good.